### ISSUE TITLE
Tag retention UI sends wrong "job_id" when updating retention policy

### ISSUE THREAD

**Issue by ywk253100** [goharbor/harbor#8722]
When I update the retention policy, the UI always sends an outdate "job_id" to the backend:
```
{"id":1,"algorithm":"or","rules":[{"id":0,"priority":0,"disabled":true,"action":"retain","template":"latestPushedK","params":{"always":30,"latestPushedK":5},"tag_selectors":[{"kind":"doublestar","decoration":"matches","pattern":"**"},{"kind":"label","decoration":"withLabels","pattern":""}],"scope_selectors":{"repository":[{"kind":"doublestar","decoration":"repoMatches","pattern":"**"}]}}],"trigger":{"kind":"Schedule","settings":{"cron":"0 0 0 * * *"},"references":{"job_id":6}},"scope":{"level":"project","ref":2},"cap":0}
```
The `job_id` sent is 6, but the actual ID isn't 6, seems that the UI caches a outdate job ID.

**Comment by ywk253100** [comment]
@AllForNothing Please contact me if you don't understand what happened, I can reproduce this issue.

**Comment by bitsf** [comment]
@ywk253100 UI just send back the id which it loads, unless you change the schedule, it will update new jobid.
What's your current case?

**Comment by bitsf** [comment]
I think what you mean is UI didn't get refresh after call backend update, so the jobid is old.

**Comment by AllForNothing** [comment]
@ywk253100 @bitsf fixed。

**Comment by AllForNothing** [comment]
fixed in PR #8727
